doc/03-Configuration.md

```diff
@@ -0,0 +1,117 @@
+# Configuration
+
+The configuration for Icinga Notifications is twofold.
+The main configuration resides in the database,
+shared between the Icinga Notifications daemon and Icinga Notifications Web.
+However, as the Icinga Notifications daemon needs to know how to access this database and some further settings,
+it needs its own configuration file as well.
+
+This configuration is stored in `/etc/icinga-notifications/config.yml`.
+See [config.example.yml](../config.example.yml) for an example configuration.
+
+## Top Level Configuration
+
+### HTTP API Configuration
+
+The HTTP API listener can be used both for submission and for debugging purposes.
+
+| Option         | Description                                                          |
+|----------------|----------------------------------------------------------------------|
+| listen         | Address to bind to, port included. (Example: `localhost:5680`)       |
+| debug-password | Password expected via HTTP Basic Authentication for debug endpoints. |
+
+### Icinga Web 2
+
+The `icingaweb2-url` is expected to point to the base directory of your Icinga Web 2 installation,
+i.e., `https://example.com/icingaweb2/`, to be used for URL creation.
+
+### Channel Plugin Directory
+
+All available Icinga Notification Channel Plugins should resident in the `channel-plugin-dir`.
+For a package install, this directory is `/usr/libexec/icinga-notifications/channels` and should not be altered.
```

Comment:
The default may differ based on the system. On Suse, it will be inside `/usr/lib/`. Apart from that, it's still singular "channel" for the time being:

https://github.com/Icinga/icinga-notifications/blob/e6f870d03de33f512f9ae8811c48c78ff3f9de11/internal/daemon/config.go#L27

Feel free to create a PR that changes this consistenly.

Reply:
> On Suse, it will be inside `/usr/lib/`.

[_[Angry Filesystem Hierarchy Standard Noises]_](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s06.html)

> Feel free to create a PR that changes this consistenly.

Will do.

Reply:
→ #225